pbf_font_tools/src/error.rs

```diff
@@ -12,4 +12,6 @@ pub enum PbfFontError {
     #[cfg(feature = "freetype")]
     #[error("Freetype error: {0}")]
     FreetypeError(#[from] crate::freetype::Error),
+    #[error("IO error: {0}")]
```

Comment:
not a biggie - just might make it a bit simpler, but totally up to you

```suggestion
    #[error(transparent)]
```

Reply:
Oh wow, I have been using `thiserror` for years and somehow missed this feature!